01 - Getting Started.qmd

```diff
@@ -203,10 +243,45 @@ penguins.drop("sex", penguins.year)
 Everything we've seen so far has been subtractive -- removing rows or columns.
 What about _adding_ columns?
 
+That's what `mutate` is for!  You can create a new column -- either as a
+function of other existing columns (for example, converting units):
+
 ```{python}
 penguins.mutate(bill_length_cm=penguins.bill_length_mm / 10)
 ```
 
+Or you can create a new column and populate it with some literal value:
```

Comment:
Small caveat is for strings which now need to be wrapped in `ibis.literal`. We changed this to match `select` behavior WRT strings.

Reply:
Not sure we need to mention it here, but maybe somewhere, since the pattern doesn't work for strings.

Reply:
oops, yeah, good call, I'll mention this